ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLog.java

```diff
@@ -505,6 +507,7 @@ public CompletableFuture<Long> onSnapshotInstalled(long lastSnapshotIndex) {
   public void close() throws IOException {
     try(AutoCloseableLock writeLock = writeLock()) {
       super.close();
+      cacheEviction.close();
       cache.close();
```

Comment:
It looks like the cache is now being called by multiple threads concurrently. Should we add more concurrency control logic inside the cache？

Reply:
The cache is thread safe and it is already called by multiple threads; see RATIS-430.

Reply:
got it